tests/proofs/dexter/dexter-properties.md

```diff
@@ -5,11 +5,11 @@ Note that the real arithmetic is used throughout the document.
 ## Storage Invariant
 
 ### Internal vs External
-- `xtzPool <= Tezos.balance`
+- `xtzPool == Tezos.balance`
 - `tokenPool <= tokenAddress.balanceOf(Tezos.self_address)`
 - `lqtTotal == lqtAddress.totalSupply()`
 
-NOTE: `<` is possible when someone "donates" XTZ or tokens (i.e., sending funds without going through `add_liquidity()`).
+NOTE: `<` is possible when someone "donates" tokens (i.e., directly sending tokens to dexter without going through any dexter entrypoints).
```

Comment:
Have we verified that it's impossible to send XTZ to a contract without triggering the `default` function? In Ethereum you can deposit funds into a contract without triggering a fallback, either by making the contract the `coinbase` of a block (and thus sending it block rewards), or by sending funds from a `selfdestruct` instruction.

Reply:
good point. @sskeirik do you have an idea?